conda/environments/cugraph_dev_cuda10.0.yml

```diff
@@ -14,10 +14,10 @@ dependencies:
 - dask-cuda=0.14*
 - dask-cudf=0.14*
 - nccl>=2.5
+- ucx-py=0.14*
```

Comment:
Is this an actual change in dependencies for cugraph? If so, the conda recipes need to be updated as well.

Reply:
Yes, and probably `ci/gpu/build.sh` around line 70 too?

Reply:
Thanks, I have added those